app/lib/methods/getUsersPresence.js

```diff
@@ -11,7 +11,7 @@ export function subscribeUsersPresence() {
 	const serverVersion = reduxStore.getState().server.version;
 
 	// if server is lower than 1.1.0
-	if (compareServerVersion(serverVersion, '1.1.0', methods.lowerThan)) {
+	if (compareServerVersion(serverVersion, 'lowerThan', '1.1.0')) {
```

Comment:
If you are not exporting the lib method, the right thing would be to use the value informed by semver and not use a "fixed" value.
This can cause future problems if change something in semver.